guide/daily-spending-wallet/backup-and-recovery/cloud-backup.md

```diff
@@ -74,28 +72,42 @@ height = 541
 %}
 
 {% include picture.html
-     image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/automatic-cloud-backup/cloud-backup-PIN.png"
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/automatic-cloud-backup/cloud-backup-PIN.png"
```

Comment:
Keypad should not include alphabet characters.

Reply:
Done